### PR TITLE
jewel: core: global/signal_handler.cc: fix typo

### PR DESCRIPTION
**Pull request by andyli029**
http://tracker.ceph.com/issues/21485

### CODE REVIEW

**Comment by joscollin** [comment]
retest this please

File: src/global/signal_handler.cc [hunk 1]
```diff
@@ -179,25 +179,22 @@ struct SignalHandler : public Thread {
   };
 
   /// all handlers
-  safe_handler *handlers[32];
+  safe_handler *handlers[32] = {nullptr};
```
Comment (joscollin) [inline src/global/signal_handler.cc]: I think the `nullptr` will be assigned only to `handlers[0]` and the remaining will be assigned just `NULL` or `0`. It's better not to put `nullptr` there or specify a range.

I mean:
`safe_handler *handlers[32] = {};`
or
`std::fill_n(handlers, 32, nullptr);`

Comment (tchaikov) [reply src/global/signal_handler.cc]: @joscollin please note this is a backport PR.

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2017-10-02_07:55:29-rados-wip-kefu-testing-2017-10-02-1347-jewel-distro-basic-mira/

**Comment by smithfarm** [comment]
This PR passed a rados integration run at http://tracker.ceph.com/issues/21742#note-14 with two bugs reproducing: 

* http://tracker.ceph.com/issues/23006
* http://tracker.ceph.com/issues/23007

@liewegas @jdurgin @tchaikov Do you think it's ready to merge?

**Pull request merged**